post_installation_configuration/index.adoc

```diff
@@ -18,25 +18,20 @@ After installing {product-title}, a cluster administrator can configure and cust
 * Alerts and notifications
 
 [id="post-install-tasks"]
-== Configuration tasks to perform after installation
+== Post-installation configuration tasks
 
-Cluster administrators can perform the following postinstallation configuration tasks:
+You can perform the following post-installation configuration tasks:
 
-* xref:../post_installation_configuration/machine-configuration-tasks.adoc#post-install-machine-configuration-tasks[Configure operating system features]:
-Machine Config Operator (MCO) manages `MachineConfig` objects. By using MCO, you can perform the following tasks on an {product-title} cluster:
+* xref:../post_installation_configuration/machine-configuration-tasks.adoc#post-install-machine-configuration-tasks[Configure operating system features]: The Machine Config Operator (MCO) manages `MachineConfig` objects. By using the MCO, you can configure nodes and custom resources.
 
-** Configure nodes by using `MachineConfig` objects
-** Configure MCO-related custom resources
+* xref:../post_installation_configuration/bare-metal-configuration.adoc#post-install-bare-metal-configuration[Configure bare metal nodes]: You can use the Bare Metal Operator (BMO) to manage bare metal hosts. The BMO can:
 
-* xref:../post_installation_configuration/bare-metal-configuration.adoc#post-install-bare-metal-configuration[Configure bare metal nodes]: The Bare Metal Operator (BMO) implements a Kubernetes API for managing bare metal hosts. It maintains an inventory of available bare metal hosts as instances of the BareMetalHost Custom Resource Definition (CRD). The Bare Metal Operator can:
-
-** Inspect the host's hardware details and report them on the corresponding BareMetalHost. This includes information about CPUs, RAM, disks, NICs, and more.
-** Inspect the host's firmware and configure BIOS settings.
+** Inspect hardware details of the host and report them to the bare metal host.
+** Inspect firmware and configure BIOS settings.
 ** Provision hosts with a desired image.
-** Clean a host's disk contents before or after provisioning.
+** Clean disk contents of the host before or after provisioning.
 
-* xref:../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[Configure cluster features]:
-As a cluster administrator, you can modify the configuration resources of the major features of an {product-title} cluster. These features include:
+* xref:../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[Configure cluster features]: You can modify the following features of an {product-title} cluster:
```

Comment:
```suggestion
* xref:../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[Configure cluster features]. You can modify the following features of an {product-title} cluster:
```

To prevent two sets of colons.